lib/folio_client/inventory.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class FolioClient
+  # Lookup items in the Folio inventory
+  class Inventory
+    attr_accessor :client
+
+    # @param client [FolioClient] the configured client
+    def initialize(client)
+      @client = client
+    end
+
+    # @param barcode [String] barcode to search by to fetch the HRID
+    def fetch_hrid(barcode:)
+      # find the instance UUID for this barcode
+      instance = client.get("/search/instances", {query: "items.barcode==#{barcode}"})
+      instance_uuid = instance.dig("instances")&.first&.dig("id")
```

Comment:
I wonder if `instance.dig("instances", 0, "id")` might work here

Reply:
nice - didn't know this syntax...presuming if there are no "instances" to dig into, it will just stop and return `nil`?

Reply:
@peetucket Yep!

Reply:
great, changed and squished